src/viewProviders/RokuAutomationViewViewProvider.ts

```diff
@@ -47,6 +48,85 @@ export class RokuAutomationViewViewProvider extends BaseRdbViewProvider {
             }
         });
 
+        this.registerCommand(VscodeCommand.rokuAutomationViewLoadAutomation, async () => {
+            if (this.isRecording) {
+                // Only allow loading when we aren't currently recording
+                await this.setIsRecording(false);
```

Comment:
Hmm. I wonder if we should show a "can't load while recording" error rather than cancelling recording?